src/simplify_rules.jl

```diff
@@ -132,10 +158,11 @@ let
     global serial_expand_simplifier
 
     function default_simplifier(; kw...)
-        IfElse(has_trig_exp,
+        IfElse(t -> (has_trig_exp(t) || has_div_rem_mod(t)),
```

Comment:
I think this needs a different branch not just `||`.

Reply:
Do you mean one branch for `&&` and two others for either one?

Reply:
No I mean that there should be another IfElse node..

`IfElse(has_div_rem_mod, Chain((divremmod_simplifier(), simplifier), simplifier)`

where simplifier is the object from this line as it was before.

Reply:
It seems weird that we would want to apply the trig simplifier if there is div mod and vice versa.

Reply:
Oh, I thought we needed both chained at this point to simplify expressions with both terms, but apparently I misunderstood it. Ok, I just made a new commit with different branches and the tests are good.